pyproject.toml

```diff
@@ -28,10 +28,6 @@ __version__ = "{version}"
 """
 
 [tool.pytest.ini_options]
-addopts = "--black --flake8 --ignore-glob=*/version.py"
-flake8-ignore = ["E133", "E203", "E226", "E228", "N802", "N803", "N806", "N812", "N813", "N815", "N816", "W503"]
-flake8-max-line-length = 110
-flake8-max-doc-length = 79
 asyncio_mode = "auto"
  
 [tool.mypy]
```

Comment:
It's save to remove mypy section as well, as that's handled by pre-commit tools.